BookingsApi/BookingsApi.IntegrationTests/BookingsApi.IntegrationTests.csproj

```diff
@@ -39,4 +39,19 @@
           <CopyToOutputDirectory>Always</CopyToOutputDirectory>
         </Content>
     </ItemGroup>
+
+    <ItemGroup>
```

Comment:
do we need these in here? none of the other feature files are?

Reply:
Thanks, my IDE put it in... DONE